docs/sphinx/install.rst

```diff
@@ -1,128 +1,183 @@
 
-You can stop and exit the container by typing the command :code:`exit`. If you did not specify
-:code:`--rm`, the container still exists as well as any changes you made in it. You can get back to it using
-the command :code:`docker start -i cuda-quantum`. 
-You can delete an existing container and any changes you made using :code:`docker rm -v cuda-quantum`. 
+.. code-block:: console
 
-Use CUDA Quantum in VS Code
-+++++++++++++++++++++++++++++++++++++
+    wget https://github.com/sylabs/singularity/releases/download/v4.0.1/singularity-ce-4.0.1.tar.gz
+    tar -xzf singularity-ce-4.0.1.tar.gz singularity-ce-4.0.1/ && rm singularity-ce-4.0.1.tar.gz && cd singularity-ce-4.0.1/
+    ./mconfig --without-suid --prefix="$HOME/.local/singularity"
+    make -C ./builddir && make -C ./builddir install && cd .. && rm -rf singularity-ce-4.0.1/
+    echo 'PATH="$PATH:$HOME/.local/singularity/bin/"' >> ~/.profile && source ~/.profile
 
-If you have `VS Code`_ installed, you can use it to work inside your container.
-To do so, install the `Dev Containers extension`_:
+For more information about using Singularity on other systems, take a look at the `admin guide <https://docs.sylabs.io/guides/4.0/admin-guide/installation.html#installation-on-windows-or-mac>`__.
 
-.. image:: _static/devContainersExtension.png 
+Once you have singularity installed, create a file `cuda-quantum.def` with the following content:
 
-Follow the steps :ref:`above<use-cuda-quantum-in-terminal>` to start the container. 
-Open VS Code and navigate to the Remote Explorer. You should see the running cuda-quantum development container listed there.
+.. code-block:: console
 
-.. image:: _static/attachToDevContainer.png 
+    Bootstrap: docker
+    From: nvcr.io/nvidia/nightly/cuda-quantum:latest
 
-Click on :code:`Attach to Container`. A new VS Code instance will open in that container. Open the `/home/cudaq`
-folder to see the `README.md` file and the CUDA Quantum examples that are included in the container. To run the examples, 
-open a terminal by going to the Terminal menu and select :code:`New Terminal`. 
+    %runscript
+        mount devpts /dev/pts -t devpts
+        cp -r /home/cudaq/* .
+        bash
 
-.. image:: _static/openTerminal.png 
+Replace the image name and/or tag in the `From` line, if necessary, with the one you want to use;
+In addition to publishing `stable releases <https://catalog.ngc.nvidia.com/orgs/nvidia/containers/cuda-quantum/tags>`__, 
+we also publish docker images whenever we update certain branches on our `GitHub repository <https://github.com/NVIDIA/cuda-quantum>`_.
```

Comment:
nit: Docker is capitalized nearly everywhere else, so it probably should be capitalized here, too.